Makefile

```diff
@@ -297,4 +297,15 @@ $(TMPDIR)/promtool:
 	oc image extract quay.io/prometheus/prometheus:main --path /bin/promtool:$(TMPDIR)/image
 	mv $(TMPDIR)/image/promtool $(TMPDIR)/promtool
 	chmod +x $(TMPDIR)/promtool
-	rm -rf $(TMPDIR)/image
\ No newline at end of file
+	rm -rf $(TMPDIR)/image
+
+$(TMPDIR)/.promoted-image-governor-kubeconfig:
+	oc --context app.ci --as system:admin --namespace ci serviceaccounts create-kubeconfig promoted-image-governor > $(TMPDIR)/.promoted-image-governor-kubeconfig
+
+go_path := $(shell go env GOPATH)
+release_folder := $(go_path)/src/github.com/openshift/release
```

Comment:
I don't think we need to know this? You can just make a path that's relative to this Makefile - look at the `local-pod-scaler` target

Reply:
Updated.